src/dataflow/models/stores/thing-store.ts

```diff
@@ -0,0 +1,34 @@
+import { types, SnapshotIn } from "mobx-state-tree";
+
+export const ThingModel = types
+  .model("Thing", {
+    thingArn: types.string,
+    thingName: types.string,
+    thingTypeName: types.maybe(types.string),
+    value: types.maybe(types.number),
+    online: true
+  })
+  .actions(self => ({
+    setValue(value: number) {
+      self.value = value;
+    }
+  }));
+export type ThingModelType = typeof ThingModel.Type;
+type ThingSnapshotType = SnapshotIn<typeof ThingModel>;
+
+export const ThingStore = types
```

Comment:
We don't use this naming pattern elsewhere in the code, but I think I prefer `ThingStore` to `Things`. The latter causes me to accidentally try things like `things[0]`, and `things.things`/`things.all`/`things.allThings` are not intuitive to me. What do you think?